package.json

```diff
@@ -41,6 +41,7 @@
     "standard-version": "^4.4.0"
   },
   "peerDependencies": {
+    "popper.js": "^1.14.7",
```

Comment:
Am I doing correctly to add popper.js as peer dependencies?

Reply:
I'm not a `vue` expert, but I don't believe so.  Making something a peer dependency means the consumer will need to provide it and they will likely not be aware of a library such as this or will be using a package like this for anything else.  This should be a full dependency because they components making use of it won't work without it.  Also, _if_ you define something as a peer dependency, it should also be defined as a dev dependency so it can be used and tested in development mode.

That said, it might be worth looking into using the `https://www.npmjs.com/package/vue-popperjs` package instead of wiring up `popper.js` directly.

Reply:
It should be a dependency.

And, as @greg-a-smith points out, It's probably work looking at the version of popperjs already prepared for vue.

Reply:
I also think that having it as a dependency is the way to go. However I think that using popper.js directly is fine if done correctly. Personally I like having the absolute minimum amount of dependencies - even if it means a little bit of extra work.

Reply:
Understand. When I saw there is no other dependency (other than dev and peer dependencies) in the fundamental-vue, that makes me think twice to add popper.js as dependency. Eventually I have to add popper.js as dependency, otherwise CI will fail me. Thanks for pointing it out.